lib/execution.ex

```diff
@@ -60,16 +57,20 @@ defmodule Firefighter.Execution do
     start(ids) |> push(id)
   end
 
-  defp to_record(%__MODULE__{event_uuid: event_uuid, elapsed: elapsed, data: data}) do
-    record =
-      %{event_uuid: event_uuid, elapsed: elapsed}
-      |> Map.merge(data)
-
-    json().encode!(record)
+  defp to_record(
+         %__MODULE__{event_uuid: event_uuid, event_time: event_time, data: data} = execution
+       ) do
+    %{
+      event_uuid: event_uuid,
+      event_time: event_time |> DateTime.to_iso8601(),
```

Comment:
This should be a stringified version/integer version of ISO time, no?

Reply:
this is a string, yes.